docs/api/rebac/groups.mdx

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 7
+title: Groups API
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+Groups API lets you manage your objects in a group. You can create, update, delete, and list groups.
```

Comment:
Groups API allows you to **add** and manage your objects in a group....

Reply:
here maybe I would mention what those objects could be: 
Users, resources, other groups etc.